src/pages/home/components/HowItWorksSection/HowItWorksSection.tsx

```diff
@@ -0,0 +1,84 @@
+import { motion } from "framer-motion"
```

Comment:
This section appears on the main page but there is also a link to `Hot it Works` in the header 🤔

Reply:
Yes, I think the page "How it works" doesn't make sense. We could remove the page and keep the section on the home page.